### ISSUE TITLE
constructor is treated as fallback in documentaion

### ISSUE THREAD

**Issue by Aniket-Engg** [OpenZeppelin/solidity-docgen#15]
In the solidity documentation, `constructor` is indexed and listed as `fallback`.

**Comment by ealmansi** [comment]
Hi @Aniket-Engg , thanks for reporting this. Could you please share a minimal example where you get this result?

**Comment by Aniket-Engg** [comment]
I wrote a token contract with this constructor:
```
constructor(string name, string symbol, uint8 decimals) ERC20Detailed(name, symbol, decimals) public{
    _mint(msg.sender, 50000 * (10 ** uint256(decimals)));
  }
```
And in the generated documentation, it was displayed as:
`<a href="https://ibb.co/dLTy50"><img src="https://thumb.ibb.co/dLTy50/soliditydoc.png" alt="soliditydoc" border="0"></a>`

**Comment by Aniket-Engg** [comment]
@ealmansi Any updates on this?

**Comment by ealmansi** [comment]
I can reproduce the bug here.

This is happening since the Solidity compiler added the `constructor` keyword in version 0.4.22 ([changes](https://medium.com/@piyopiyo/how-to-write-constructor-in-solidity-version-0-4-22-and-above-dcb8c042974f)), which solidity-docgen cannot yet handle correctly. Up to 0.4.21, a function definition with an empty name was always a fallback function, but now constructors also pass this test.

It should not be too hard to update this project to be able to handle the new feature since the compiler outputs an `isConstructor` property on all function definitions, and therefore it's easy to distinguish between fallback and constructor functions when rendering the docs for a given function.

**Comment by ajsantander** [comment]
@frangio 
We can reproduce the bug as well in the ZeppelinOS documentation: https://docs.zeppelinos.org/docs/upgradeability_AdminUpgradeabilityProxy.html#

If you click on fallback, the page jumps to a weird location. And, as stated above the fallback entry actually shows the constructor function of the contract.

**Comment by frangio** [comment]
This is fixed in v0.2 (still in alpha). See #23 to learn more about it.
